Mathlib/Topology/Algebra/InfiniteSum/Basic.lean

```diff
@@ -638,6 +642,28 @@ theorem tsum_range {g : γ → β} (f : β → α) (hg : Injective g) :
   simp_rw [← comp_apply (g := g), tsum_univ (f ∘ g)]
 #align tsum_range tsum_range
 
+open Set in
+/-- If `f b = 0` for all `b ∈ t`, then the sum over `f a` with `a ∈ s` is the same as the
+sum over `f a` with `a ∈ a ∖ t`. -/
+lemma tsum_eq_tsum_diff [T2Space α] {f : β → α} (s t : Set β) (hf₀ : ∀ b ∈ t, f b = 0) :
```

Comment:
```suggestion
lemma tsum_setElem_eq_tsum_setElem_diff [T2Space α] {f : β → α} (s t : Set β) (hf₀ : ∀ b ∈ t, f b = 0) :
```
since `a : s` refers to `Set.Elem`. `setCoe` might also be reasonable.

Reply:
I think you can use [Function.Injective.hasSum_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/InfiniteSum/Basic.html#Function.Injective.hasSum_iff) with g = [Set.inclusion](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Basic.html#Set.inclusion).

Reply:
OK to the name change.


> I think you can use [Function.Injective.hasSum_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/InfiniteSum/Basic.html#Function.Injective.hasSum_iff) with g = [Set.inclusion](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Basic.html#Set.inclusion).

The `tsum` version does not assume summability, so using `Function.Injective.hasSum_iff` would require a case distinction regarding `Summable f`. Not sure if this is preferable.